test/plugin/test_out_detect_exceptions.rb

```diff
@@ -125,7 +127,7 @@ def test_ignore_nested_exceptions
     }
 
     test_cases.each do |language, exception|
-      cfg = "languages #{language}"
+      cfg = "#{CONFIG}\nlanguages #{language}"
```

Comment:
Might be easier to read as:
```ruby
        cfg = %{
#{CONFIG}
languages #{language}}
```
or, for consistency with the exception traces:
```ruby
        cfg = <<END
#{CONFIG}
languages #{language}
END
```
instead of the explicit `\n`s. Also below. WDYT?

Reply:
Yeah I agree. Thanks for the suggestion. Done.